fuse/dl/models/backbones/backbone_mlp.py

```diff
@@ -46,7 +46,9 @@ def __init__(
 
         self.mlp = nn.ModuleList(mlp_layers)
 
-    def forward(self, input_tensor: Dict) -> Dict:
+    def forward(self, input_tensor: torch.Tensor) -> torch.Tensor:
+        input_tensor = input_tensor.float()
```

Comment:
@mosheraboh 

I'm not sure that's the right practice. Feels fishy.
I had to use it in order to supports both the MLP and the Transformer with the same DTI dataset.

What do you think?

Reply:
Yes,
What was the dtype before you've changed it?
Let's check which code generated this tensor.

Reply:
@mosheraboh 
I'm casting it to `int` explicitly in the dataset using:
```
(OpToTensor(), dict(key=["data.drug.encoding", "data.target.encoding"], dtype=torch.int32)),
```

Do we expect dataset to return `float` all the time? If so lets delete this change and I'll take care of it in my DTI pipeline 😃

Reply:
The input to the model should be float.
Not sure if that's what you asked.
Anyway, I think we should remove this line.